12/umbraco-cms/tutorials/custom-error-page.md

```diff
@@ -238,7 +238,7 @@ If you set up everything correctly and the error pages are not showing correctly
 
 * Custom [ContentFinders](../reference/routing/request-pipeline/icontentfinder.md) in your solution,
 * Any packages that allow you to customize redirects, or
-* Rewrite rules in web.config that might interefere with custom error handling.
+* Rewrite rules in web.config that might interfere with custom error handling.
```

Comment:
to keep the docs consistent, can you please make this change to v10 and v11 as well?